src/NATS.Client.KeyValueStore/NatsClientExtensions.cs

```diff
@@ -0,0 +1,31 @@
+using NATS.Client.Core;
+using NATS.Client.JetStream;
+
+namespace NATS.Client.KeyValueStore;
+
+public static class NatsClientExtensions
+{
+    /// <summary>
+    /// Creates a NATS Key-Value Store context using the specified NATS client.
+    /// </summary>
+    /// <param name="client">The NATS client instance.</param>
+    /// <returns>An instance of <see cref="INatsKVContext"/> which can be used to interact with the Key-Value Store.</returns>
+    public static INatsKVContext CreateKeyValueStoreContext(this INatsClient client)
+        => CreateKeyValueStoreContext(client.Connection);
```

Comment:
Does it make sense to chain these to the JS extensions?

```cs
public static INatsKVContext CreateKeyValueStoreContext(this INatsClient client)
    => CreateKeyValueStoreContext(client.CreateJetStreamContext());

public static INatsKVContext CreateKeyValueStoreContext(this INatsConnection connection)
    => CreateKeyValueStoreContext(connection.CreateJetStreamContext());
```

Reply:
makes sense 💯